string_width/string_width.ts

```diff
@@ -0,0 +1,23 @@
+// Copyright 2018-2023 the Deno authors. All rights reserved. MIT license.
+
+import { stripAnsi } from "https://deno.land/std@$STD_VERSION/string_width/_strip_ansi.ts";
+import { unicodeWidth } from "https://deno.land/std@$STD_VERSION/string_width/_unicode_width.ts";
```

Comment:
Use relative module specifiers (like `./_unicode_width.ts`) in the runtime code of the module. (`https://deno.land/std@$STD_VERSION` should be used only in the examples in docs.)